recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
```

Comment:
Also we need the re-render with 2.2.1+ (2.2.2 would be best) as it fixes a bug on new AppVeyor images.